documentation/adr/0002-wiki-platform.md

```diff
@@ -0,0 +1,294 @@
+- [GitHub Wiki](gh-wiki)
+- [GitBook](gitbook)
+- [WikiJS](wiki-js)
+- Git-based Headless CMS (e.g. [Decap CMS](git-cms-decap) and [Tina.io](git-cms-tina))
+- API-based Headless CMS (e.g. [Strapi](strapi) and [Directus](directus))
+
+## Decision Outcome <!-- REQUIRED -->
+
+Chosen option: "{option 1}", because {justification. e.g., only option which meets k.o. criterion decision driver | which satisfies x condition | ... }.
+
+### Positive Consequences <!-- OPTIONAL -->
+
+- {e.g., improved performance on quality metric, new capability enabled, ...}
+- ...
+
+### Negative Consequences <!-- OPTIONAL -->
+
+- {e.g., decreased performance on quality metric, risk, follow-up decisions required, ...}
+- ...
+
+## Pros and Cons of the Options <!-- OPTIONAL -->
+
+### Confluence
+
+[Confluence](confluence) is a Software as a Service (SaaS) documentation and collaboration tool offered by Atlassian that organizes content into "spaces" and offers a series of templates and components that can be used to create custom documentation for internal and external stakeholders.
+
+#### Details
+
+- **Hosting:** SaaS
+- **Pricing:** [$5.75 (standard) or $11 (premium) per user per month](confluence-pricing)
+- **Content Review:** Supported through drafts, not enforceable
+- **Supported Media:** TODO
+- **I18n:** Limited third party plugins for automating translation
+- **Web Analytics:** [Google Analytics plugin](confluence-ga) available, also native analytics with premium tier
+- **Open Source Status:** Propietary
+- **External Contributions:** Only supported in public spaces (without review)
+
+#### Pros
+
+- Relatively user friendly for non-technical users
+- Supports a wide variety of media and page content
+- Supports drafts, which allow edits to be made without publishing
+- Supports page history and comparison of previous versions
+- Limited support for public pages and spaces
+- Minimimal ongoing maintenance costs due to SaaS hosting
+- Most affordable per user cost for a given tier of features
+
+#### Cons
+
+- Does not support a formal review process for drafts before they can be published
+- Supports fewer content types than Notion, especially in terms of structured data
+- Content can only be made public at the level of an individual page or an entire space
+- Content API is limited in capability and less intuitive than Notion API
+- No direct support for internationalization and localization
+- Closed source proprietary tool
+- Data is controlled by Confluence, only accessible via API
+
+### Notion
+
+[Notion](notion) is a Software as a Service (SaaS) documentation and collaboration tool that also allows users to add structured and semi-structured content to pages. Because Notion offers a fully-featured API for reading and managing content, it also has a robust set of community integrations that extend Notion's core functionality.
+
+#### Details
+
+- **Hosting:** SaaS
+- **Pricing:** [$8 (pro) or $15(business) per user per month](notion-pricing)
```

Comment:
same pricing question

Reply:
Same note as above. All prices are for named users with edit access to the wiki.